vllm/v1/worker/gpu_model_runner.py

```diff
@@ -955,6 +973,15 @@ def _prepare_inputs(
                     scheduler_output.
                     num_common_prefix_blocks[kv_cache_group_id])
 
+                if isinstance(kv_cache_group_spec.kv_cache_spec,
```

Comment:
```
        encoder_seq_lens: Optional[np.ndarray] = None
        if isinstance(kv_cache_group_spec.kv_cache_spec, CrossAttentionSpec):
            encoder_seq_lens= self._prepare_encoder_seq_lens(necessary_args)
```
Can you put this part of code into a seperate function like above?

And I slightly prefer to put this under the for loop than to put it under the `if isinstance(EncoderOnlyAttentionSpec): xxx; else xxx` block.